symphony/app/fbcnms-projects/magmalte/app/components/lte/LteDashboard.js

```diff
@@ -151,50 +178,34 @@ function LteNetworkDashboard({startEnd}: {startEnd: [moment, moment]}) {
 
   return (
     <div className={classes.dashboardRoot}>
-      <Grid container spacing={3}>
+      <Grid container spacing={4}>
         <Grid item xs={12}>
-          <Paper>
-            <EventAlertChart startEnd={startEnd} />
-          </Paper>
+          <EventAlertChart startEnd={startEnd} />
         </Grid>
 
         <Grid item xs={12}>
           <DashboardAlertTable />
         </Grid>
 
-        <Grid item xs={6}>
-          <Paper elevation={2}>
-            <GatewayKPIs />
-          </Paper>
-        </Grid>
-
-        <Grid item xs={6}>
-          <Paper elevation={2}>
-            <EnodebKPIs />
-          </Paper>
-        </Grid>
-
         <Grid item xs={12}>
-          <Text>
-            <GpsFixed /> Events (388)
-          </Text>
-          <Paper>
-            <div className={classes.contentPlaceholder}>
-              Events Table Goes Here
-            </div>
-          </Paper>
+          <DashboardKPIs />
         </Grid>
       </Grid>
     </div>
   );
 }
 
-function NetworkDashboardTabLabel() {
+function DashboardTabLabel(props) {
   const classes = useStyles();
 
   return (
     <div className={classes.tabLabel}>
-      <NetworkCheck className={classes.tabIconLabel} /> Network
+      {props.label === 'Subscribers' ? (
+        <People className={classes.tabIconLabel} />
+      ) : props.label === 'Network' ? (
+        <NetworkCheck className={classes.tabIconLabel} />
+      ) : null}
+      {props.label}
```

Comment:
I think it's better to pass in the icon, instead of making this chaining of conditions to determine the icon based on labels :)

Reply:
Similar response to the `DashboardKPIs.js`, but I'm not sure how to pass a component like that via props. Happy to make the change though once I know.

Reply:
```
function MyComponent(props) {
  const Icon = props.icon;
  return
    <div>
      <Icon />
      {props.label}
    </div>;
}
...

return (
  <MyCompnent icon={NetworkCheck} label="Network">
);
```